src/utils/sources-tree/utils.js

```diff
@@ -34,7 +34,7 @@ export function isInvalidUrl(url: Object, source: Object) {
   return (
     IGNORED_URLS.indexOf(url) != -1 ||
     !source.get("url") ||
-    source.get("loading") ||
+    source.loadedState === "loading" ||
```

Comment:
Tried this, but during the initial load, the loadedState value is always in "unloaded" state. Hence, this condition always passes and no source is shown in the source tree. Do we need to change the loadedState to "loaded" somewhere?